extension/src/openvic-extension/classes/GUILabel.cpp

```diff
@@ -157,46 +154,29 @@ void GUILabel::_notification(int what) {
 				if (string_segment == nullptr) {
 					if (currency_texture.is_valid()) {
 						currency_texture->draw(
-							ci, position - Vector2 {
-								1.0_real, static_cast<real_t>(currency_texture->get_height()) * 0.75_real
-							}
+							ci, position - Vector2 { 1.0_real, static_cast<real_t>(currency_texture->get_height()) * 0.75_real }
 						);
 						position.x += currency_texture->get_width();
 					}
 				} else {
 					font->draw_string(
-						ci, position, string_segment->text, HORIZONTAL_ALIGNMENT_LEFT, -1, font_size,
-						string_segment->colour
+						ci, position, string_segment->text, HORIZONTAL_ALIGNMENT_LEFT, -1, font_size, string_segment->colour
 					);
 					position.x += string_segment->width;
 				}
 			}
 
 			position.y += font->get_descent(font_size);
 		}
-
 	} break;
 	}
 }
 
 GUILabel::GUILabel()
-  : tooltip_active { false },
-	gui_text { nullptr },
-	text {},
-	substitution_dict {},
-	horizontal_alignment { HORIZONTAL_ALIGNMENT_LEFT },
-	max_size {},
-	border_size {},
-	adjusted_rect {},
-	auto_adjust_to_content_size { false },
-	font {},
-	font_size { DEFAULT_FONT_SIZE },
-	default_colour {},
-	colour_codes { nullptr },
-	currency_texture {},
-	background {},
-	lines {},
-	line_update_queued { false } {}
+	: tooltip_active { false }, gui_text { nullptr }, text {}, substitution_dict {},
+	  horizontal_alignment { HORIZONTAL_ALIGNMENT_LEFT }, max_size {}, border_size {}, adjusted_rect {},
+	  auto_adjust_to_content_size { false }, font {}, font_size { DEFAULT_FONT_SIZE }, default_colour {},
+	  colour_codes { nullptr }, currency_texture {}, background {}, lines {}, line_update_queued { false } {}
```

Comment:
It's much easier to read long initialisation lists when each argument has its own line, especially when checking if something is missing, or when adding or removing entries.